v1/api/swagger/paths/hosts@{hostId}@reimage.yaml

```
@@ -0,0 +1,32 @@
+summary: Reimage a specific host
+post:
+  tags:
+    - hosts
+  summary: Reimage Host by ID
+  description: >-
+    Re-deploys a host to the associated machine.
```

"to the same machine" might be more clear?
```suggestion
    Re-deploys a host to the same machine.
```